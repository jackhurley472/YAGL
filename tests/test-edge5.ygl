
Graph team;

Node adam("Adam");
Node james("James");
Node tank("Tank");
Node jack("Jack");

team + adam;
team + james;
team + tank;
team + jack;

team: adam -> james -> tank -> jack -> adam;

printGraph(team);


