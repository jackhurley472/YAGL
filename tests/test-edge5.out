============== Graph Print ===============
	Stats: 	4	4	4

Nodes:
Node (0) : Adam --- Node (1) : James --- Node (2) : Tank
Node (3) : Jack

Edges:
[(0) : Adam] ---(1)---> [(1) : James]
[(1) : James] ---(1)---> [(2) : Tank]
[(2) : Tank] ---(1)---> [(3) : Jack]
[(3) : Jack] ---(1)---> [(0) : Adam]
============ End Graph Print =============
